content/tutorials/getting-started/index.md

```diff
@@ -166,60 +175,113 @@ Now, you can directly skip to the [generate a package]({{< ref "#generate-packag
-GATLING_HOME is set to .../gatling-charts-highcharts-bundle-{{< var gatlingVersion >}}
-GATLING_VERSION is set to '{{< var gatlingVersion >}}'
-Creating jar...done
+$ ./bin/gatling.sh --api-token <API_TOKEN>
 ```
 
-This script can also be clicked or double clicked from any file explorer. You'll see the result in the folder `target`, which should now have a file named `package.jar`.
-
-{{< youtube _fq3giAIibw >}}
+```
+Creating the package
+Package created
+Do you want to create a new simulation or start an existing one?
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Create a new Simulation on Gatling Enterprise, then start it
+[2] Start an existing Simulation on Gatling Enterprise
+```
 
-#### Step 3.2 - Upload the Package {#upload-package}
+Let's create a new simulation, type `1`
 
-On Gatling Enterprise, click on the [Packages section]({{< ref "../../reference/user/package_conf/" >}}) and on create. Give a name to your package, you can choose to make it available to all teams or to limit access to a specific one, then hit save.
+```
+Proceeding to the create simulation step
+Choose one simulation class in your package:
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] computerdatabase.ComputerDatabaseSimulation
+```
 
-Next, upload the package you generated earlier by clicking on the cloud icon. The package you generated earlier is located here: `target/package.jar`
+Select your Simulation class (the only available here) with `1`
 
-Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+```
+Choose a team from the list:
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Team 'Default team', id='b236af88-1c24-4623-b872-cb8993651c15'
+```
 
-#### Step 3.3 - Create a Simulation {#create-simulation}
+Select a special team, or the default one for your organization with `1`. This will allow access to the simulation to users within this team.
 
-To get a grasp of how to run a simulation with Gatling, you will find a ready to use script in the bundle. So let's get started with your first simulation!
+```
+Enter a simulation name, or just hit enter to accept the default name (ComputerDatabaseSimulation)
+Waiting for user input...
+```
 
-In the [Simulations section]({{< ref "../../reference/user/simulations/" >}}), click on **Create**.
+Enter the name of your simulation, then hit **Enter**
 
-Choose a name for your simulation and define the team that will have access to this simulation (if none has been created yet, a default one named **Default team** exist).
+```
+Do you want to create a new package or upload your project to an existing one?
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Create a new package on Gatling Enterprise
+[2] Choose an existing package on Gatling Enterprise
+```
+The package is just the jar file containing your simulation class.
```

Comment:
Don't mention jar files, non-java users don't know what it is.
"Tha package is an archive containing your simulation classes."